sdcm/sct_config.py

```diff
@@ -294,7 +293,12 @@ class SCTConfiguration(dict):
              help="""If True, all backtraces found in db nodes would be decoded automatically"""),
 
         dict(name="instance_provision", env="SCT_INSTANCE_PROVISION", type=str,
-             help="instance_provision: on_demand|spot_fleet|spot_low_price|spot_duration"),
+             help="instance_provision: spot|on_demand|spot_fleet"),
```

Comment:
I thought we agreed to have spot_duration and low_price as options as well but "hide" them.

Reply:
Yes, it's hidden.
```
 help="instance_provision: spot|on_demand|spot_fleet"),
```
where is the problem here?